misc/docs/source/tutorial/code_snippets/stationxml_file_from_scratch.rst

```diff
@@ -0,0 +1,89 @@
+=============================================================================
+Creating a StationXML file from Scratch
+=============================================================================
+
+Creating a custom StationXML file is a task that sometimes comes up in
+seismology. This section demonstrates how to it with ObsPy. Please note that
+this is not necessarily easier or more obvious then directly editing an XML
+file but it does provider tighter integration with the rest of ObsPy and can
+guarantee a valid result at the end.
+
+Note that this assumes a certain familiarity with the `FDSN StationXML standard
+<https://www.fdsn.org/xml/station/>`_. We'll create a fairly simplistic
+StationXML file and many arguments are optional. ObsPy will validate the
+resulting StationXML file against its schema upon writing so the final file is
+assured to be valid against the StationXML schema.
+
+The following illustration shows the basic structure of ObsPy's internal
+representation.
+
+.. figure:: /_images/Inventory.png
+
+Each big box will be an object and all objects will have to be hierarchically
+linked to form a single :class:`~obspy.core.inventory.inventory.Inventory`
+object. An inventory can contain any number of
+:class:`~obspy.core.inventory.network.Network` objects, which in turn can
+contain any number of :class:`~obspy.core.inventory.station.Station` objects,
+which once again in turn can contain any number of
+:class:`~obspy.core.inventory.channel.Channel` objects. Instrument
```

Comment:
This is great.  A suggestion: a mention that response information is not covered might be more useful right at the top (even in the title).  When a users sees the tutorial title about creating StationXML they, like me, might imagine dealing with the hard part of responses.

Reply:
See #1899